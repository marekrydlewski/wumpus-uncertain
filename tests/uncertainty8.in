10 10
0.3
??????????
??????????
??????????
B?????????
OB????????
OOBB??????
OOOOB?????
OOOB??????
OOB???????
OOOB??????
