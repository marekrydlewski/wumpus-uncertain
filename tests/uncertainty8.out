0.30 0.30 0.30 0.30 0.30 0.30 0.30 0.30 0.30 0.30 
0.30 0.30 0.30 0.30 0.30 0.30 0.30 0.30 0.30 0.30 
0.59 0.30 0.30 0.30 0.30 0.30 0.30 0.30 0.30 0.30 
0.00 0.59 0.30 0.30 0.30 0.30 0.30 0.30 0.30 0.30 
0.00 0.00 1.00 0.48 0.30 0.30 0.30 0.30 0.30 0.30 
0.00 0.00 0.00 0.00 0.74 0.30 0.30 0.30 0.30 0.30 
0.00 0.00 0.00 0.00 0.00 0.38 0.30 0.30 0.30 0.30 
0.00 0.00 0.00 0.00 0.38 0.30 0.30 0.30 0.30 0.30 
0.00 0.00 0.00 1.00 0.30 0.30 0.30 0.30 0.30 0.30 
0.00 0.00 0.00 0.00 0.30 0.30 0.30 0.30 0.30 0.30 
